sdk/identity/azure-identity/azure/identity/_credentials/cli_credential.py

```diff
@@ -0,0 +1,44 @@
+# ------------------------------------
```

Comment:
Note: If we want to merge it back to master eventually, we'll need some testing, likely by mock subprocess to return the JSON we know we get from CLI.